ports/renesas-ra/boards/ARDUINO_PORTENTA_C33/ra_cfg/fsp_cfg/r_dac_cfg.h

```diff
@@ -0,0 +1,13 @@
+/* generated configuration header file - do not edit */
```

Comment:
Following how the other renesas-ra boards are done, please move these 5 files (r_dac_cfg.h  r_sce_cfg.h  r_usb_basic_cfg.h  r_usb_class_cfg.h  r_usb_pcdc_cfg.h) to `ports/renesas-ra/fsp_cfg/`.